### PR TITLE
fix: Repos with long names get cut off

### LINKED ISSUE #1750: Bug: Repos with long names get cut off
### Describe the bug

Repos with longer names get cut off when viewing an insight page:

Screenshot 2023-09-26 at 11 59 01 AM

This makes for a tricky user experience: I'm really not sure _exactly_ which repos are which.

### Steps to reproduce

1. Create an insight page with repo names that are long
2. See the insight page and notice the names getting cut off 

CC @isabensusan - more feedback 😅 nothing urgent, just a sharp edge

### Browsers

_No response_

### Additional context (Is this in dev or production?)

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [X] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by aryanas159**
## Description
Truncating repo names using css instead of javascript.

fixes #1750 
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Mobile & Desktop Screenshots/Recordings
**Before -** 
Truncating string programmatically.
image

**After -** 
Using CSS to truncate the string.
Screenshot 2023-09-29 223545

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by nickytonline** [comment]
Thanks for the PR @aryanas159. Can you add a before and after screenshot of with a repository with a long name? You can always change the text manually in the component to a long name to demonstrate the fix if you don't have a repository with a long name.

Thanks!

**Comment by aryanas159** [comment]
> Thanks for the PR @aryanas159. Can you add a before and after screenshot of with a repository with a long name? You can always change the text manually in the component to a long name to demonstrate the fix if you don't have a repository with a long name.
> 
> Thanks!

@nickytonline  I have added the before and after screenshots.

File: components/molecules/TableRepositoryName/table-repository-name.tsx [hunk 1]
```diff
@@ -27,9 +26,12 @@ const TableRepositoryName = ({ avatarURL, fullName }: TableRepositoryNameProps):
 
       {/* Text */}
       <div className="flex flex-col justify-center">
-        <div title={name} className="  text-base text-light-slate-12 tracking-tight">
+        <div
+          title={name}
+          className="  text-base text-light-slate-12 tracking-tight max-w-[85px] sm:max-w-[150px] md:max-w-[110px] whitespace-nowrap overflow-hidden overflow-ellipsis"
```
Comment (nickytonline) [inline components/molecules/TableRepositoryName/table-repository-name.tsx:31]: The fix looks good, just not sure what widths we want with ellpipsis enabled.

Comment (OgDev-01) [inline components/molecules/TableRepositoryName/table-repository-name.tsx:31]: if you must use arbitrary value for sizes... use the `rem` equivalent 
```suggestion
          className="  text-base text-light-slate-12 tracking-tight max-w-[5.5rem] sm:max-w-[9.3rem] md:max-w-[7rem] whitespace-nowrap overflow-hidden overflow-ellipsis"
```

**Review by OgDev-01 — CHANGES_REQUESTED** [review]

File: components/molecules/TableRepositoryName/table-repository-name.tsx (hunk 1 above)
Comment (nickytonline) [reply components/molecules/TableRepositoryName/table-repository-name.tsx]: @isabensusan, @OgDev-01, are these the standard widths for the app for this kind of field, or are these values arbitrary?

Comment (isabensusan) [reply components/molecules/TableRepositoryName/table-repository-name.tsx]: hey! those are arbitrary, no idea where they come from

Comment (OgDev-01) [reply components/molecules/TableRepositoryName/table-repository-name.tsx]: as far as the table goes, i think this width works... we've been using arbitrary values to specify the widths

**Review by OgDev-01 — CHANGES_REQUESTED** [review]

File: components/molecules/TableRepositoryName/table-repository-name.tsx [hunk 2]
```diff
@@ -27,9 +26,12 @@ const TableRepositoryName = ({ avatarURL, fullName }: TableRepositoryNameProps):
 
       {/* Text */}
       <div className="flex flex-col justify-center">
-        <div title={name} className="  text-base text-light-slate-12 tracking-tight">
+        <div
+          title={name}
```
[the remark this answers is not in the document]
Comment (aryanas159) [reply components/molecules/TableRepositoryName/table-repository-name.tsx]: @OgDev-01  I have added the tooltip, this is how it looks.
image

**Review by isabensusan — APPROVED** [review]

**Review by OgDev-01 — APPROVED** [review]

**Pull request merged**